### DISCUSSION TITLE
property inspector : 3d transform info always visible ?

### DISCUSSION THREAD

**Discussion by AndHalfAFish** [godotengine/godot-proposals#4317] [Editor]
I think the property inspector space could be used a bit better. For instance, for an Area node : 

![property_inspector](https://user-images.githubusercontent.com/59080934/161207383-8a350477-96c8-4ff5-b20c-d9342647c80f.png)

I wonder if the 2 orange areas are really used so frequently that they deserve to be always visible ? Personally, I would prefer it if the transform info would be open by default, because that info is probably used to most by people (also for animations etc).

**Comment by lentsius-bark** [comment]
I agree, I go into the transform properties far too often always wishing it was either at the top, or open by default.
The way I see it is that with objects, in a game, their position/rotation and scale are key and used most often.

**Comment by Calinou** [comment]
See also https://github.com/godotengine/godot-proposals/issues/4274. I personally wouldn't unfold the Transform section by default, as it's quite tall. Vertical screen real estate is precious :slightly_smiling_face:

**Comment by AndHalfAFish** [comment]
I guess pinning would be a solution for most people's needs ... 
And maybe combine that with the possibility of minimizing/closing every 'submenu' (which is not the case now) ?

**Reply by Calinou** [reply]
> And maybe combine that with the possibility of minimizing/closing every 'submenu' (which is not the case now) ?

I'm not sure if allowing the folding of object sections (such as CollisionObject or Spatial on your example) is worth it. Usability tends to suffer a lot when multiple levels of folding are used, so it should be avoided whenever possible.

As for properties at the top and bottom, they aren't in any section, so they can't be folded. In your example, there are only 3 properties at the top and they're often used, so I don't think folding them is worth it. I also don't see much point in folding the properties at the bottom, since they will be out of view automatically if your screen doesn't have enough vertical space available anyway.
